section4/exercises/classProperties.js

```diff
@@ -34,10 +64,53 @@ Make sure to run the file with node in your command line.
 
 // Prompt 1: Dog
 
+class Dog {
+constructor() {
+  this.weight = weight;
+  this.breed = breed;
+  this.color = color;
```

Comment:
🚨 Does this actually work? If you are expecting `weight` to be a parameter, where else should it appear?